code/modules/pai/pai.dm

```diff
@@ -334,7 +334,10 @@
 	master_name = "The Syndicate"
 	master_dna = "Untraceable Signature"
 	// Sets supplemental directive to this
-	laws.supplied[1] = "Do not interfere with the operations of the Syndicate."
+	add_supplied_law(0, "Do not interfere with the operations of the Syndicate.")
+	qdel(leash) // Freedom!!!
```

Comment:
QDEL_NULL

Reply:
that reminds me i dont think i qdel_null this on destroy either :P

Reply:
then make sure stuff can handle it being null

Reply:
Applied with https://github.com/tgstation/tgstation/pull/77325/commits/54a276db6ad271ec5157dbada5c4b1a9ec5c876f